Arrowgene.Ddon.GameServer/Handler/CharacterEditUpdatePawnEditParamExHandler.cs

```diff
@@ -25,6 +26,28 @@ public override void Handle(GameClient client, StructurePacket<C2SCharacterEditU
             Server.Database.UpdateEditInfo(pawn);
             pawn.Name = packet.Structure.Name;
             Server.Database.UpdatePawnBaseInfo(pawn);
+
+            //Client won't let you reincarnate if you're wearing a gender-locked item, but EquipmentTemplates also have to be cleaned.
```

Comment:
If this is duplicate code from the other handler it should be moved to its own function.